packages/@aws-cdk/aws-codedeploy/lib/lambda/deployment-group.ts

```diff
@@ -0,0 +1,196 @@
+  autoRollback?: AutoRollbackConfig;
+}
+export class LambdaDeploymentGroup extends cdk.Construct implements ILambdaDeploymentGroup {
+  /**
+   * Import an Lambda Deployment Group defined either outside the CDK,
+   * or in a different CDK Stack and exported using the {@link #export} method.
+   *
+   * @param parent the parent Construct for this new Construct
+   * @param id the logical ID of this new Construct
+   * @param props the properties of the referenced Deployment Group
+   * @returns a Construct representing a reference to an existing Deployment Group
+   */
+  public static import(scope: cdk.Construct, id: string, props: LambdaDeploymentGroupImportProps): ILambdaDeploymentGroup {
+    return new ImportedLambdaDeploymentGroup(scope, id, props);
+  }
+
+  public readonly application: ILambdaApplication;
+  public readonly deploymentGroupName: string;
+  public readonly deploymentGroupArn: string;
+
+  constructor(scope: cdk.Construct, id: string, props: LambdaDeploymentGroupProps) {
+    super(scope, id);
+
+    let serviceRole: iam.Role | undefined = props.serviceRole;
+    if (serviceRole) {
+      if (serviceRole.assumeRolePolicy) {
+        serviceRole.assumeRolePolicy.addStatement(new iam.PolicyStatement()
+          .addAction('sts:AssumeRole')
+          .addServicePrincipal('codedeploy.amazonaws.com'));
+      }
+    } else {
+      serviceRole = new iam.Role(this, 'ServiceRole', {
+        assumedBy: new iam.ServicePrincipal('codedeploy.amazonaws.com')
+      });
+    }
+    serviceRole.addToPolicy(new iam.PolicyStatement()
+      .addResource('arn:aws:s3:::*/CodeDeploy/*')
+      .addActions('s3:GetObject', 's3:GetObjectVersion'));
+    serviceRole.addToPolicy(new iam.PolicyStatement()
+      .addResource('*')
+      .addCondition('StringEquals', {
+        's3:ExistingObjectTag/UseWithCodeDeploy': 'true'
+      })
+      .addActions('s3:GetObject', 's3:GetObjectVersion'));
+    serviceRole.addToPolicy(new iam.PolicyStatement()
+      .addResource(props.alias.functionArn)
+      .addActions('lambda:UpdateAlias', 'lambda:GetAlias'));
+    if (props.alarms) {
+      serviceRole.addToPolicy(new iam.PolicyStatement()
+        .addResources(...props.alarms.map(alarm => alarm.alarmArn))
+        .addAction('cloudwatch:DescribeAlarms'));
+    }
+
+    const alarms = props.alarms || [];
+
+    const resource = new CfnDeploymentGroup(this, 'Resource', {
+      applicationName: props.application.applicationName,
+      serviceRoleArn: serviceRole.roleArn,
+      deploymentGroupName: props.deploymentGroupName,
+      deploymentConfigName: `CodeDeployDefault.Lambda${props.deploymentConfig}`,
+      deploymentStyle: {
+        deploymentType: props.deploymentConfig === LambdaDeploymentConfig.AllAtOnce ?
+          DeploymentType.InPlace : DeploymentType.BlueGreen,
+        deploymentOption: props.deploymentConfig === LambdaDeploymentConfig.AllAtOnce ?
+          DeploymentOption.WithoutTrafficControl : DeploymentOption.WithTrafficControl
```

Comment:
I don't think these values make sense. The only valid values for `deploymentStyle` for Lambda are:

```
      deploymentStyle: {
        deploymentType: 'BLUE_GREEN',
        deploymentOption: 'WITH_TRAFFIC_CONTROL',
      },
```